.drone.star

```diff
@@ -0,0 +1,80 @@
+OC_CI_PHP = "owncloudci/php:%s"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GITHUB_RELEASE = "plugins/github-release"
+
+DEFAULT_PHP_VERSION = "8.1"
+
+config = {
+    "branches": [
+        "main",
+    ],
+    "codestyle": True,
+    "phpstan": True,
+    "phan": True,
+    "phpunit": True,
+}
+
+trigger = {
+    "ref": [
+        "refs/head/main",
```

Comment:
I think, we want to run CI in PRs, main-push and tags not for push to the main

Reply:
yes, drone CI does not seem to have run for the latest commit(s). So this trigger needs to include more things.